scanners/zap/tests/test_setup_podman.py

```diff
@@ -171,6 +171,26 @@ def test_setup_ajax(test_config):
             break
 
 
+def test_setup_graphql(test_config):
+    TEST_GRAPHQL_ENDPOINT = "http://test.com/graphql"
+    TEST_GRAPHQL_SCHEMA_URL = "http://test.com/schema.graphql"
+    TEST_GRAPHQL_SCHEMA_FILE = "schema.graphql"
+
+    test_config.set("scanners.zap.graphql.endpoint", TEST_GRAPHQL_ENDPOINT)
+    test_config.set("scanners.zap.graphql.schemaUrl", TEST_GRAPHQL_SCHEMA_URL)
+    test_config.set("scanners.zap.graphql.schemaFile", TEST_GRAPHQL_SCHEMA_FILE)
```

Comment:
Because of path translation (see below), this will not work. 
Also, because the file needs to be actively copied, it needs to exist.

I'd suggest using : 
```
test_config.set("scanners.zap.graphql.schemaFile" = __file__
```
i.e.: the python script itself, this way we dont need to create a file, and we are sure that it points to something, it's the same trick as for the `scanners.zap.importUrlsFromFile` test